<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="11057" started-at="2017-01-11T14:39:46Z" finished-at="2017-01-11T14:39:57Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="11057" started-at="2017-01-11T14:39:46Z" finished-at="2017-01-11T14:39:57Z">
      <class name="testcases.AddEmployee$AddEmployeeTest">
        <test-method status="PASS" signature="initialize(java.lang.String)[pri:0, instance:testcases.AddEmployee$AddEmployeeTest@482f8f11]" name="initialize" is-config="true" duration-ms="6172" started-at="2017-01-11T06:39:46Z" finished-at="2017-01-11T06:39:52Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialize -->
        <test-method status="FAIL" signature="AddNoDiscount()[pri:0, instance:testcases.AddEmployee$AddEmployeeTest@482f8f11]" name="AddNoDiscount" duration-ms="3482" started-at="2017-01-11T06:39:52Z" finished-at="2017-01-11T06:39:56Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Error found in dashboard data after adding employee without discount expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Error found in dashboard data after adding employee without discount expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at testcases.AddEmployee$AddEmployeeTest.AddNoDiscount(AddEmployee.java:17)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AddNoDiscount -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:testcases.AddEmployee$AddEmployeeTest@482f8f11]" name="tearDown" is-config="true" duration-ms="1377" started-at="2017-01-11T06:39:56Z" finished-at="2017-01-11T06:39:57Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testcases.AddEmployee$AddEmployeeTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
